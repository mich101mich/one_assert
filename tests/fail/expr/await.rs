fn main() {
    let fut = async { 1 };
    let _ = async move {
        one_assert::assert!(fut.await);
    };
}
