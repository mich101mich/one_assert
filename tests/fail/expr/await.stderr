error[E0308]: mismatched types
 --> tests/fail/expr/await.rs:4:29
  |
4 |         one_assert::assert!(fut.await);
  |                             ^^^^^^^^^ expected `bool`, found integer
